packages/workbox-core/src/controllers/WorkboxCore.js

```diff
@@ -0,0 +1,20 @@
+import assert from '../utils/assert';
+import logHelper from '../utils/logHelper';
+
+export default class WorkboxCore {
+  get INTERNAL() {
+    if (this._internal) {
+      return this._internal;
+    }
+
+    this._internal = {
+      logHelper,
+    };
+
+    if (process.env.NODE_ENV !== 'production') {
+      this._internal.assert = assert;
+    }
```

Comment:
Test is here: https://github.com/GoogleChrome/workbox/blob/e7ac90b92664ecb5a2e357df474660497ab65346/packages/workbox-core/test/bundle/node/controllers/test-WorkboxCore.js#L27